### DISCUSSION TITLE
UDM-P: Factory resetting?

### DISCUSSION THREAD

**Discussion by tjcrandley** [unifi-utilities/unifi-common#203] [Q&A] [answered]
I messed up with the installation of the UDM Utilities, and now get an error when trying to run “podman version”.  Instead of trying to fix my mistake I want to factory reset the UDM-P and start again (I learned a lot the first attempt).  If I follow the factory rest steps, will that remove all my UDM Utilities work?

**Comment by tjcrandley** [comment, marked as answer]
Was told elsewhere that the factory reset will remove all the UDM Utilities changes and start you fresh.

**Reply by Ryamonster10** [reply]
yes it will
